ext/storage_sources/s3_store/CMakeLists.txt

```diff
@@ -18,7 +18,7 @@ target_link_libraries(
 )
 
 # Build the S3 extension as a library.
-set(sources "s3_storage_source.cpp")
+set(sources "s3_storage_source.cpp" "S3StoreLogSystem.cxx")
```

Comment:
Should we change the extension to `.cpp`?

Reply:
Yes - as we decided it is all .cpp now.